docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+phpdoc:
+  container_name: phpdoc
+  image: phpdoc:latest
+  environment:
+    SYMFONY_DEBUG: "true"
+    SYMFONY_ENV: "dev"
+    SYMFONY__KERNEL__CACHE_DIR: "/tmp/cache"
+    SYMFONY__KERNEL__LOGS_DIR: "/tmp/logs"
+  volumes:
+      - /opt/webapp
+      - ./docker/vhost.conf:/etc/apache2/sites-available/000-default.conf
+      - ./docker/development:/etc/apache2/sites-available/development.override
+  working_dir: "/opt/webapp"
+  ports: [ "8000:80" ]
+  links:
+    - mysql
+
+mysql:
```

Comment:
can you name it database please? That will make it easier to integrate it into rancher since I tend to give the services a technology agnostic name (as much as possible)